docs/how_to_use_csctl_plugin_openstack.md

```diff
@@ -0,0 +1,61 @@
+# Using the csctl plugin for OpenStack
+
+## What does the csctl plugin for OpenStack do?
+
+As a user, you can create clusters based on Cluster Stacks with the help of the Cluster Stack Operators (CSO and CSPO). The operators need certain files, such as those required to apply the necessary Helm charts and to obtain information about the versions in the cluster stack.
+
+To avoid generating these files manually, you can use [CSCTL](https://github.com/SovereignCloudStack/csctl). In the case of provider-specific Cluster Stacks, the `CSCTL` tool invokes the provider-specific CSCTL plugin. Therefore, the CSCTL plugin for OpenStack is essential if the user intends to build, upload node images to an S3 bucket, and then import them into Glance.
+
+## Different methods of csctl plugin for OpenStack
```

Comment:
How and where can be these methods configured?

Reply:
I think this can be pretty easily seen in an example, but I can add a reference to `csctl.yaml` to make it even more clear for newcomers.

Reply:
See https://github.com/SovereignCloudStack/csctl-plugin-openstack/pull/6/files#r1607797389, e.g. in a get method I am missing that I need `node-images` folder with `config.yaml` which is only copied into the release.

Reply:
You have a similar sentence in the `get method` section, but it does not specify that `config.yaml` must be in the `node-images` folder. However, this can be inferred from the example folder structure. But yes, I can add that `config.yaml` must be in `node-images` folder.